src/nvim/ex_docmd.c

```diff
@@ -6816,7 +6840,7 @@ void post_chdir(int local)
 
 
 /*
- * ":cd", ":lcd", ":chdir" and ":lchdir".
+ * ":cd", ":tcd", ":lcd", ":chdir", ":tchdir" and ":lchdir".
```

Comment:
I have no idea how you managed to pass clint test (should break because multiline comment starts on the previous line and clint complains about “complex multiline comments”), but this needs to be transformed into `///`.